R/summarize.R

```diff
@@ -0,0 +1,84 @@
+#' summarize WQ site information
+#'
+#' @return data frame of the following information:
+#' @export
+#' @param sites character ID(s) of sites to summarize
+#' @param siteInfo data frame containing site information, with column named CODIGO_ESTACAO 
+#' containing station id's
+#' @param nutriDF data frame record of nutrient measurements
+#' @importFrom dplyr filter 
+#' @importFrom dplyr mutate
+#' @importFrom dplyr bind_cols
+summarizeSites <- function(sites, siteInfo, nutriDF){
+  #get existing site metadata
+  summaryDF <- filter(siteInfo, CODIGO_ESTACAO %in% sites)
```

Comment:
i think we decided to only summarize one site at a time, in which case this filtering step is no longer necessary at all, right?